auth_proxy/models/oauth.py

```diff
@@ -68,7 +70,9 @@ class Grant(Base):
     _scopes = Column('scopes', Text)
 
     def delete(self):
-        self.expires = 0
+        self.expires = datetime.now()
```

Comment:
It's something called inside flask-oauthlib, https://github.com/lepture/flask-oauthlib/blob/0cabc05073daefbba126eb2c8f2004e4d2a561b7/flask_oauthlib/provider/oauth2.py#L748, and there isn't really a way to hook into it.